Formula/ltex-ls.rb

```diff
@@ -0,0 +1,41 @@
+class LtexLs < Formula
+  desc "LSP for LanguageTool with support for Latex, Markdown and Others"
+  homepage "https://valentjn.github.io/ltex/"
+  url "https://github.com/valentjn/ltex-ls/archive/refs/tags/15.2.0.tar.gz"
+  sha256 "59209730cb9cda57756a5d52c6af459f026ca72c63488dee3cfd232e4cfbf70a"
+  license "MPL-2.0"
+  head "https://github.com/valentjn/ltex-ls.git", branch: "develop"
+
+  depends_on "maven" => :build
+  depends_on "python@3.9" => :build
+  depends_on "openjdk@11"
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_prefix
```

Comment:
it seems this relies on a `python` executable being available: https://github.com/valentjn/ltex-ls/blob/4954220ebd34df6c032ca646fd9bab5bb4d8231e/pom.xml#L376

we can add one to `PATH`:
```suggestion
    ENV.prepend_path "PATH", Formula["python@3.9"].opt_libexec/"bin"
    ENV["JAVA_HOME"] = Formula["openjdk@11"].opt_prefix
```